draft-opsarea-rfc5706bis.md

```diff
@@ -53,6 +53,16 @@ author:
     fullname: Ran Chen
     organization: ZTE
     email: chen.ran@zte.com.cn
+
+normative:
+
+informative:
+
+  CHECKLIST:
+    title: Operations and Management Review Checklist
+    date: 2025
+    target: https://raw.githubusercontent.com/IETF-OPS-DIR/Review-Template/refs/heads/main/review-checklist.md
```

Comment:
Rather than the raw, would it be better to publish this to a gist or a GitHub Page so it can be viewed with formatting?